openedx/core/djangoapps/user_api/accounts/utils.py

```diff
@@ -176,3 +178,17 @@ def retrieve_last_sitewide_block_completed(username):
         course_key=text_type(item.location.course_key),
         location=text_type(item.location),
     )
+
+
+def generate_password(length=12, chars=string.letters + string.digits):
```

Comment:
You're really not taking advantage of the full range of Unicode here, Bill. Get some emojis in these passwords!